setup.py

```diff
@@ -60,7 +60,7 @@ def read(*rnames):
         'zope.dottedname',
         'zope.event',
         'zope.interface',
-        'zope.schema',
```

Comment:
This comma can stay, I think also black wants it.

Reply:
It seems that the whole package is not black-ified, I added the comma, but I didn't applied the other black suggestions.

Reply:
@ale-it @sneridagh let's black-ify the package after merging this PR and before doing the next release. Maybe it makes sense to set up a gh actions job that just runs static code analysis on Plone core packages.

Reply:
Sorry! I did not mean to run black!
I only noticed that the comma was removed (with no reason) and did not make sense to me to merge that chunk because sooner or later it will be reverted.
Long story short I think the PR is fine now :)